cl/opinion_page/templates/includes/opinion_tabs_content.html

```diff
@@ -115,14 +115,23 @@ <h3 class="bottom serif">
 {% elif tab == "cited-by" %}
     {# Cited By #}
     <div class="tab-pane fade in active" id="cited-by">
-        <h2 class="opinion-section-title jump-link">
-            <span>
-                Cited By
-                <a href="/feed/search/?type=o&q=cites%3A({{ cluster.sub_opinions.all|OR_join }})" rel="nofollow">
-                    <i class="gray fa fa-rss" title="Subscribe to a feed of citations to this case."></i>
+        <div class="row">
+            <div class="col-xs-6">
+                <h2 class="opinion-section-title jump-link">
+                    <span>
+                        Cited By
+                        <a href="/feed/search/?type=o&q=cites%3A({{ cluster.sub_opinions.all|OR_join }})" rel="nofollow">
+                            <i class="gray fa fa-rss" title="Subscribe to a feed of citations to this case."></i>
+                        </a>
+                    </span>
+                </h2>
+            </div>
+            <div class="col-xs-6 text-right">
+                <a href="/?q=cites%3A({{ cluster.sub_opinions.all|OR_join }})" rel="nofollow" class="btn btn-default opinion-section-title">
+                    View All Citing Opinions
```

Comment:
Maybe: 

```suggestion
                    Filter Citing Opinions
```

To be more consistent?

Reply:
it seems good to me

Reply:
Let's do it, and then I think we can merge?

Reply:
yes, i pushed the changes a few minutes ago